docs/README.md

```diff
@@ -0,0 +1,52 @@
+# Documentation README
+Check out the docs at http://hyperledger-indy.readthedocs.io/projects/plenum
+
+This `docs/` directory should contain all of the documents associated with the repository, with the exception of README files that describe technical details with the directories of code. 
+
+The `docs/source/` directory contains all of the documentation that is built and hosted on readthedocs. Based on the maintainer's discretion, some docs may not be necessary to build on readthedocs, and can exist within the `docs/` directory but outside of the `source/` directory. 
+
+## How to Add Documentation
+For new features and pull requests, maintainers should make sure that the **contributor has added an explanation for their changes in the docs folder before merging the PR.**
+  
+Contributors should write an addition to a current file or add a new file to the docs/source/ folder that explains what their feature is and how it works. If needed, they may also add a link to more technical README's located nearer to the code.
+
+Whenever additions are made to the docs, make sure to update the `index.rst` in whichever folder the file has been added, and build the docs locally to confirm they work (TODO: add the `sphinx-build` command to our CI/CD flow).
```

Comment:
Do I understand correctly that for now updating the docs on `readthedocs` is a manual process, and we expect it to be part of CI/CD soon? Who is responsible for updating the `readthedocs` while it's not automated?